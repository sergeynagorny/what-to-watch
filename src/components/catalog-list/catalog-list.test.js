import React from "react";
import renderer from "react-test-renderer";
import CatalogList from "./catalog-list.jsx";


const films = [
  {
    id: 1,
    title: `What We Do in the Shadows`,
    poster: `img/what-we-do-in-the-shadows.jpg`,
  },
  {
    id: 2,
    title: `Pulp Fiction`,
    poster: `img/pulp-fiction.jpg`,
  }
];

it(`Should CatalogList render correctly`, () => {
  const tree = renderer.create(
      <CatalogList
        films={films}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
