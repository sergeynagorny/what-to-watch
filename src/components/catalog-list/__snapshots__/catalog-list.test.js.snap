// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CatalogList render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        poster="img/what-we-do-in-the-shadows.jpg"
        width="280"
      >
        <source />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        What We Do in the Shadows
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        poster="img/pulp-fiction.jpg"
        width="280"
      >
        <source />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Pulp Fiction
      </a>
    </h3>
  </article>
</div>
`;
